hr_loan_management/models/hr_loan.py

```diff
@@ -0,0 +1,776 @@
+            ],
+        },
+    )
+    request_date = fields.Date(
+        string="Realization Request Date",
+        required=True,
+        default=fields.Date.today(),
+        readonly=True,
+        states={
+            "draft": [
+                ("readonly", False),
+            ],
+        },
+    )
+    date_realization = fields.Date(
+        string="Realization Date",
+        readonly=True,
+        states={
+            "draft": [
+                ("readonly", False),
+            ],
+        },
+    )
+    loan_type_id = fields.Many2one(
+        string="Loan Type",
+        comodel_name="hr.loan.type",
+        required=True,
+        readonly=True,
+        states={
+            "draft": [
+                ("readonly", False),
+            ],
+        },
+    )
+    currency_id = fields.Many2one(
+        string="Currency",
+        related="loan_type_id.currency_id",
+        store=True,
+        readonly=True,
+    )
+    loan_amount = fields.Float(
+        string="Loan Amount",
+        required=True,
+        readonly=True,
+        states={
+            "draft": [
+                ("readonly", False),
+            ],
+        },
+        digits=dp.get_precision("Account"),
+    )
+    maximum_loan_amount = fields.Float(
+        string="Maximum Loan Amount",
+        related="loan_type_id.maximum_loan_amount",
+        store=True,
+        readonly=True,
+        digits=dp.get_precision("Account"),
+    )
+    interest = fields.Float(
+        string="Interest (p.a)",
+        related="loan_type_id.interest_amount",
+        store=True,
+        readonly=True,
+        required=True,
+        states={
+            "draft": [
+                ("readonly", False),
+            ],
+        },
+        digits=dp.get_precision("Account"),
+    )
+    maximum_installment_period = fields.Integer(
+        string="Maximum Installment Period",
+        related="loan_type_id.maximum_installment_period",
+        store=True,
+        readonly=True,
+    )
+    manual_loan_period = fields.Integer(
+        string="Loan Period",
+        readonly=True,
+        states={
+            "draft": [
+                ("readonly", False),
+            ],
+        },
+        required=True,
+    )
+    date_payment = fields.Selection(
+        string="Date Payment",
+        selection=DATE_SELECTION,
+        required=True,
+    )
+    first_payment_date = fields.Date(
+        string="First Payment Date",
+        compute="_compute_first_payment_date",
+        readonly=True,
+        store=True,
+    )
+    total_principle_amount = fields.Float(
+        string="Total Principle Amount",
+        compute="_compute_total",
+        store=True,
+        digits=dp.get_precision("Account"),
+    )
+    total_interest_amount = fields.Float(
+        string="Total Interest Amount",
+        compute="_compute_total",
+        store=True,
+        digits=dp.get_precision("Account"),
+    )
+    realized = fields.Boolean(
+        string="Realized",
```

Comment:
Realized means that loan has been given to employee. I will give a tool tip on my next commit